### ISSUE TITLE
Dependencies Error

### ISSUE THREAD

**Issue by sbc640964** [ikenfin/vite-plugin-sentry#47]
![image](https://user-images.githubusercontent.com/46646044/181246300-2613e2b0-a4e5-43c0-a095-91f1dbd65a35.png)

**Comment by ikenfin** [comment]
Hi!
Yes, it seems we have too high minimal versions for peer dependencies in package:( 

I will fix it asap, as workaround for now you can run installer in legacy mode with `--legacy-peer-deps` to skip version checks

**Comment by ikenfin** [comment]
@sbc640964 hi! Just released 1.1.3 to npm, can you try it?

**Comment by olivervorasai** [comment]
Hi I just ran into this when upgrading my dependencies. 

This was my previous state:
```
Package                    Current   Wanted  Latest  Location                                Depended by
vite                        2.9.13   2.9.14   3.0.3  node_modules/vite                       client
vite-plugin-sentry           1.1.0    1.1.3   1.1.3  node_modules/vite-plugin-sentry         client
```

Using `npm upgrade` causes it to fail. But upgrading Vite to v3 first and then upgrading vite-plugin-sentry after works fine.

**Comment by ikenfin** [comment]
@olivervorasai 
Hi! It’s strange, because 1.1.3 has softer peer deps version range (vite 2.6+ or 3.0+)🤔
I will test upgrade case too

**Comment by sbc640964** [comment]
@ikenfin 
![image](https://user-images.githubusercontent.com/46646044/181703896-14c4fa86-d372-4806-bffe-d8c718afcf2b.png)

**Comment by ikenfin** [comment]
@sbc640964 hi! It seems I’ve found the issue. Nprm contains wrong package.json version, because at build/publish ci state, yarn replaces conditional peer dependency version with latest vite 3.x before it become sent to npm. So when you run npm install it seems that minimal required vite version is 3.0.3 and fails because you using 2.x. I will try to fix it with mixing yarn and npm —no-install.

**Comment by ikenfin** [comment]
@sbc640964 @olivervorasai Seems that fixed now in 1.1.4 👍
